src/Encamina.Enmarcha.SemanticKernel.Plugins.QuestionAnswering/Plugins/QuestionAnsweringPlugin.cs

```diff
@@ -127,31 +130,42 @@ public QuestionAnsweringPlugin(Kernel kernel, string modelName, Func<string, int
     /// </summary>
     /// <param name="input">The question to answer with information from a context given in <paramref name="context"/>.</param>
     /// <param name="context">The context with information that may contain the answer for question from <paramref name="input"/>.</param>
+    /// <param name="locale">The locale in which the response is generated. This parameter is optional. If not provided, the <paramref name="input"/> language is used.</param>
     /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests.</param>
     /// <returns>A string representing the answer for the <paramref name="input"/> based on all the information found from searching the memory's collections.</returns>
     [KernelFunction]
     [Description(@"Answer questions using information from a context.")]
     public virtual async Task<string> QuestionAnsweringFromContextAsync(
         [Description(@"The question to answer with information from a context.")] string input,
         [Description(@"Context with information that may contain the answer for question")] string context,
+        [Description(@"The locale in which the response is generated. This parameter is optional. If not provided, the input language is used.")] string locale = null,
         CancellationToken cancellationToken = default)
     {
         var functionArguments = new KernelArguments(questionAnsweringFromContextFunctionExecutionSettings)
         {
             [@"input"] = input,
             [@"context"] = context,
+            [@"locale"] = GetAnswerLocale(locale),
         };
 
         var functionResult = await kernel.InvokePromptAsync(QuestionAnsweringFromContextFunctionPrompt, functionArguments, cancellationToken: cancellationToken);
 
         return functionResult.GetValue<string>();
     }
 
-    private Task<int> GetQuestionAnsweringFromContextFunctionUsedTokensAsync(KernelFunction questionAnsweringFromContextFunction, string input, CancellationToken cancellationToken)
+    private static string GetAnswerLocale(string locale)
+    {
+        return string.IsNullOrEmpty(locale)
```

Comment:
Instead of `IsNullOrEmpty` it would be better to use `IsNullOrWhitespace`.

Reply:
Fixed!